apps/src/levelbuilder/lesson-editor/ActivityCard.jsx

```diff
@@ -151,34 +152,44 @@ class ActivityCard extends Component {
         </div>
         <div style={styles.activityBody} hidden={this.props.collapsed}>
           {activity.activitySections.map(section => (
-            <ActivitySectionCard
-              key={section.key}
-              activitySection={section}
-              activityPosition={activity.position}
-              activitySectionsCount={activity.activitySections.length}
-              activitiesCount={this.props.activitiesCount}
-              ref={ref => {
-                setActivitySectionRef(ref, activity.position, section.position);
-              }}
-              activitySectionMetrics={this.props.activitySectionMetrics}
-              updateTargetActivitySection={updateTargetActivitySection}
-              clearTargetActivitySection={clearTargetActivitySection}
-              targetActivityPos={this.props.targetActivityPos}
-              targetActivitySectionPos={this.props.targetActivitySectionPos}
-              updateActivitySectionMetrics={updateActivitySectionMetrics}
-              hasLessonPlan={hasLessonPlan}
-              allowMajorCurriculumChanges={allowMajorCurriculumChanges}
-            />
+            <div key={section.key + '-container'}>
```

Comment:
Instead of making the DOM deeper and a little harder to follow with an extra nesting div, consider using a React fragment like so:
```
  <>
    <ActivitySectionCard>...</ActivitySectionCard>
    <div key={section.key + 'add'}></div>
  </>
```

If it warns you about missing a key for each child in the list or whatever (I'm not 100% sure it does warn with newer react versions, it might suffice to have keys for both child elements), instead of the `<>` you can also use the more explicit syntax:

```
<React.Fragment key={section.key + '-fragment'>
    <ActivitySectionCard>...</ActivitySectionCard>
    <div key={section.key + 'add'}></div>
</React.Fragment>
```

(`<></>` is also great to use when you want to make a component that includes more than one top-level DOM element)